### PR TITLE
Mark ramp/slope tiles as unbuildable.

### PR DESCRIPTION
**Pull request by phrohdoh**
Works in RA2.

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/Buildings/BuildingUtils.cs [hunk 1]
```diff
@@ -27,6 +27,12 @@ public static bool IsCellBuildable(this World world, CPos cell, BuildingInfo bi,
 			if (!bi.AllowInvalidPlacement && world.ActorMap.GetUnitsAt(cell).Any(a => a != toIgnore))
 				return false;
 
+			var tile = world.Map.MapTiles.Value[cell];
+			var rampType = world.TileSet.Templates[tile.Type][tile.Index].RampType;
```
Comment (Mailaender) [inline OpenRA.Mods.Common/Traits/Buildings/BuildingUtils.cs]: I suggest

```csharp
var rampType = world.TileSet.GetTileInfo(tile).RampType;
```

instead.

**Comment by Mailaender** [comment]
:+1: This works well in TS, too.

File: OpenRA.Mods.Common/Traits/Buildings/BuildingUtils.cs (hunk 1 above)
Comment (Phrohdoh) [inline OpenRA.Mods.Common/Traits/Buildings/BuildingUtils.cs]: Done.

**Comment by Phrohdoh** [comment]
Updated.

**Comment by penev92** [comment]
Works :+1:

**Comment by Phrohdoh** [comment]
#7650 is a fix for an NRE caused by this.

**Pull request merged**